backend/skeleton.toml

```diff
@@ -7,3 +7,5 @@ password=
 [ocp-server]
 port=8000
 
+[jira]
+host_url=
```

Comment:
This `host_url` doesn't match the one used in the `JiraService` class, and the `personal_access_token` is missing.